maps/src/main/res/layout/offline_map_layers_picker.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is missing a new line at the end. Make sure to enable automatic line breaks in Android Studio to avoid this (Settings > Editor > General > On Save > Ensure every saved file ends with a line break).

Reply:
Done.